.github/workflows/build.yml

```diff
@@ -5,9 +5,17 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - '**.md'
+      - 'LICENSE'
+      - 'images/**'
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - '**.md'
```

Comment:
is there a way to have an ignore file and include / reference it in multiple places? If not, seems like a good change...

Reply:
That would be pretty sweet.  I haven't been able to find anything 😞

Reply:
oh well, too bad. Merge this in then and if we find something we can switch at that point